File: 151004/Gonzarevich/lab4/publisher/DTO/IssueResponseTo.php
<?php

namespace App\DTO;

use App\Entity\Issue;

class IssueResponseTo
{
    public readonly int $id;
    public readonly int $userId;
    public readonly string $title;
    public readonly string $content;

    public function __construct(
        Issue $issue
    ) {
        $this->id = $issue->getId();
        $this->userId = $issue->getUserId()->getId();
        $this->title = $issue->getTitle();
        $this->content = $issue->getContent();
    }
}

File: 151001/Trubchik/dc/publisher/src/main/java/by/bsuir/ilya/dto/IssueResponseTo.java
package by.bsuir.ilya.dto;

import java.sql.Date;
import java.time.LocalDateTime;

public class IssueResponseTo {
    private long id;

    private long userId;

    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;

    public void setId(long id) {
        this.id = id;
    }

    public long getId() {
        return id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public LocalDateTime getCreated() {
        return created;
    }

    public LocalDateTime getModified() {
        return modified;
    }

    public long getUserId() {
        return userId;
    }

    public String getTitle() {
        return title;
    }

    public void setCreated(LocalDateTime created) {
        this.created = created;
    }

    public void setModified(LocalDateTime modified) {
        this.modified = modified;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setUserId(long userId) {
        this.userId = userId;
    }
}
